openacademy/reports.xml

```diff
@@ -0,0 +1,32 @@
+<odoo>
+
+    <report
```

Comment:
Rename this file please something like `openacademy_session_report.xml` and rename it in the manifest too.